composer.json

```
@@ -0,0 +1,43 @@
+{
+    "name": "spryker-sdk/upgrader",
+    "type": "library",
+    "description": "Upgrade spryker vendor.",
+    "license": "proprietary",
+    "require": {
+        "php": ">=7.3",
+        "ext-json": "*",
+        "ergebnis/json-printer": "^3.1",
+        "symfony/console": "^5.0",
+        "symfony/options-resolver": "^5.0",
+        "symfony/process": "^5.0"
+    },
+    "require-dev": {
+        "codeception/codeception": "^4.1",
+        "codeception/module-asserts": "^1.1",
+        "codeception/module-phpbrowser": "^1.0.0",
+        "spryker/code-sniffer": "^0.15.6",
+        "phpstan/phpstan": "^0.12.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "Upgrader\\": "src/Upgrader/"
+        }
+    },
+    "bin": [
+        "bin/upgrader"
+    ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "extra": {
+        "branch-alias": {
+            "dev-master": "1.0.x-dev"
+        }
+    },
+    "config": {
+        "sort-packages": true
+    },
+    "scripts": {
+        "stan": "phpstan analyze -l 5 src/",
```

```suggestion
        "stan": "phpstan analyze",
```
as you neon file contains already everything.